frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -138,6 +138,7 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
             </Col>
             <Col className="map-filter-typeahead">
               <AsyncTypeahead
+                disabled={findMoreOpen}
```

Comment:
I just noticed that this new field wasn't disabling when the search window was open so I fixed it real quick